# Readings

Please try to read the first paper for each week. Other papers listed will be discussed in class and/or will be helpful for the homeworks. In cases where the Pubmed of the paper does not have links to OpenAccess versions of the paper, we will provide a pdf (requires being logged in to Hopkins SSO).

### Week 1: scRNA-seq data generation

1. [Macosko EZ, Basu A, Satija R, et al. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell. 2015;161(5):1202-1214. doi:10.1016/j.cell.2015.05.002](https://pubmed.ncbi.nlm.nih.gov/26000488/)

2. [Zheng GX, Terry JM, Belgrader P, et al. Massively parallel digital transcriptional profiling of single cells. Nat Commun. 2017;8:14049. Published 2017 Jan 16. doi:10.1038/ncomms14049](https://pubmed.ncbi.nlm.nih.gov/28091601/)

### Week 2: scRNA-seq analysis

1. [Quality control](https://www.sc-best-practices.org/preprocessing_visualization/quality_control.html) sections: 6.0 - 6.3
2. [Normalization](https://www.sc-best-practices.org/preprocessing_visualization/normalization.html) section 7.0 - 7.1
3. [PCA](https://www.sc-best-practices.org/preprocessing_visualization/dimensionality_reduction.html) 9.0 - 9.1
4. [Clustering](https://www.sc-best-practices.org/cellular_structure/clustering.html) Whole chapter
5. [Manual cell typing](https://www.sc-best-practices.org/cellular_structure/annotation.html) 11.0 - 11.4
 

### Week 3: Integration and cell typing

#### Typing and integration methods
1. [Kiselev VY, Yiu A, Hemberg M. scmap: projection of single-cell RNA-seq data across data sets. Nat Methods. 2018 May;15(5):359-362. doi: 10.1038/nmeth.4644](https://pubmed.ncbi.nlm.nih.gov/29608555/)
2. [Pola≈Ñski K, Young MD, Miao Z, Meyer KB, Teichmann SA, Park JE. BBKNN: fast batch alignment of single cell transcriptomes. Bioinformatics. 2020 Feb 1;36(3):964-965. doi: 10.1093/bioinformatics/btz625](https://pubmed.ncbi.nlm.nih.gov/31400197/)
3. [Tan Y, Cahan P. SingleCellNet: A Computational Tool to Classify Single Cell RNA-Seq Data Across Platforms and Across Species. Cell Syst. 2019 Aug 28;9(2):207-213.e2. doi: 10.1016/j.cels.2019.06.004. Epub 2019 Jul 31](https://pubmed.ncbi.nlm.nih.gov/31377170/)
3. [Korsunsky I, Millard N, Fan J, Slowikowski K, Zhang F, Wei K, Baglaenko Y, Brenner M, Loh PR, Raychaudhuri S. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat Methods. 2019 Dec;16(12):1289-1296. doi: 10.1038/s41592-019-0619-0](https://pubmed.ncbi.nlm.nih.gov/31740819/)

#### Benchmarking studes
1. [Tran HTN, Ang KS, Chevrier M, Zhang X, Lee NYS, Goh M, Chen J. A benchmark of batch-effect correction methods for single-cell RNA sequencing data. Genome Biol. 2020 Jan 16;21(1):12. doi: 10.1186/s13059-019-1850-9](https://pubmed.ncbi.nlm.nih.gov/31948481/)
2. [Kang JB, Nathan A, Weinand K, Zhang F, Millard N, Rumker L, Moody DB, Korsunsky I, Raychaudhuri S. Efficient and precise single-cell reference atlas mapping with Symphony. Nat Commun. 2021 Oct 7;12(1):5890. doi: 10.1038/s41467-021-25957-x](https://pubmed.ncbi.nlm.nih.gov/34620862/)
3. [Open problems in Single-cell analysis](https://openproblems.bio/results/)




