# Readings

Please try to read the first paper for each week. Other papers listed will be discussed in class and/or will be helpful for the homeworks. In cases where the Pubmed of the paper does not have links to OpenAccess versions of the paper, we will provide a pdf (requires being logged in to Hopkins SSO).

### Week 1: scRNA-seq data generation

1. [Macosko EZ, Basu A, Satija R, et al. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell. 2015;161(5):1202-1214. doi:10.1016/j.cell.2015.05.002](https://pubmed.ncbi.nlm.nih.gov/26000488/)

2. [Zheng GX, Terry JM, Belgrader P, et al. Massively parallel digital transcriptional profiling of single cells. Nat Commun. 2017;8:14049. Published 2017 Jan 16. doi:10.1038/ncomms14049](https://pubmed.ncbi.nlm.nih.gov/28091601/)

### Week 2: scRNA-seq analysis

1. [Quality control](https://www.sc-best-practices.org/preprocessing_visualization/quality_control.html) sections: 6.0 - 6.3

2. [Normalizatin](https://www.sc-best-practices.org/preprocessing_visualization/normalization.html) section 7.0 - 7.1

3. [PCA](https://www.sc-best-practices.org/preprocessing_visualization/dimensionality_reduction.html) 9.0 - 9.1

4. [Clustering](https://www.sc-best-practices.org/cellular_structure/clustering.html) Whole chapter

5. [Manual cell typing](https://www.sc-best-practices.org/cellular_structure/annotation.html) 11.0 - 11.4
 

